src/hooks/useUniversalRouter.ts

```diff
@@ -50,9 +50,8 @@ export function useUniversalRouterSwapCallback(
       try {
         gasEstimate = await provider.estimateGas(tx)
       } catch (gasError) {
-        await provider.call(tx) // this should throw the actual error
-        // If the actual error is not thrown, just try again:
-        gasEstimate = await provider.estimateGas(tx)
+        console.warn(gasError)
```

Comment:
is this the preexisting behavior or are we adding a new console log ? if new, do you think it's necessary?